sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.3.0-beta.1 (Unreleased)
-
+### New Features
+- Added `AdtQueryBuilder` to allow for the creation of ADT queries using fluent-style syntax to abstract the ADT query language from users. As of 6-28-21, this is a trial feature that will be recieving feedback before potentially being shipped in the future.
```

Comment:
I don't think you need to have a timestamp in here.

Reply:
Consider a date format that doesn't confuse the rest of the world. I'd suggest 2021-06-28 or more generic like July 2021.